src/api/data.ts

```diff
@@ -0,0 +1,34 @@
+import { User } from '../types/User';
+import { Post } from '../types/Post';
+import { Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+
+export const getUsers = () => {
+  return client.get<User[]>('/users');
+}; // return userList
+
+export const getUserPosts = (userId: number) => {
+  return client.get<Post[]>(`/posts?userId=${userId}`);
+}; // return list of user posts
+
+export const getComment = (postId: number) => {
```

Comment:
```suggestion
export const getComments = (postId: number) => {
```
Actually, you are getting an array of comments. not a singular comment

Reply:
Done